hooks/tk-houdini_actions.py

```diff
@@ -231,7 +231,9 @@ def _file_cop(self, path, sg_publish_data):
         # remove non alphanumeric characers from the string (houdini node names
         # must be alphanumeric). first, build a regex to match non alpha-numeric
         # characters. Then use it to replace any matches with an underscore
-        pattern = re.compile('[\W_]+', re.UNICODE)
+        
+        # cannot use special characters to create nodes
+        pattern = re.compile('[\W_]+')
```

Comment:
this change looks right to me - `path` should always be a utf-8 encoded string, so the `re.UNICODE` directive that got removed didn't make any sense!

Reply:
looks good. Unrelated, I wish this code would do something like `achmédèád.jpg` => `achmedead.jpg` instead of `achm_d_.jpg`